spaces/S000168/README.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: S000168
+name: Real projective plane
+aliases:
+  - $\mathbb RP^2$
+  - $P_2(\mathbb R)$
+  - RP2
+---
+
+The quotient of a {S169} $S^2$ identifying its antipodal points.
```

Comment:
```suggestion
The quotient of a {S169} $S^2$ identifying its antipodal points.

Equivalently, the quotient of $[0,1]^2$ identifying $\langle t,0\rangle$ with $\langle 1-t,1\rangle$ and identifying $\langle 0,t\rangle$ with $\langle 1,1-t\rangle$.
```

The latter characterization will be good to compare with Mobius strip, Klein bottle, and torus.

Reply:
Would a representation as a quotient of $[-1,1]^2$ be easier?  Then the identification is just "taking the negative" ($(t,1)$ identified with $(-t,-1)$, etc).  But then it's basically the two dimensional disk slightly deformed with opposite points on the circumference identified.  That seems a cleaner and nicer formulation, and actually one of the presentations suggested by @GeoffreySangston.

Reply:
Lee's Topological Manifolds Proposition 6.2 has a slick formulation of the square version: 
- $\\{(x, y) : |x| + |y| \le 1\\}/\sim$, where $\sim$ is the minimal equivalence relation such that $(x,y) \sim (-x, -y)$ when $|x| + |y| = 1$.

Though if that seems gimmicky, it could instead be:

- $[-1,1]^2/\sim$, where $\sim$ is the minimal equivalence relation such that $p \sim -p$ for boundary points $p$.

It seems desirable to include the Grassmannian version (maybe just calling it the space of lines through the origin), since that's the definition someone might have in mind, and also the homogeneous coordinates version for the same reason. I think the hemi-sphere is fine to omit, and either the square or disk could be omitted.

Reply:
I am fine with using $[-1,1]^2$ in place of $[0,1]^2$.

Reply:
> The latter characterization will be good to compare with Möbius strip, Klein bottle, and torus.

Nice. I'll consider adding Möbius strip and Klein bottle, etc.

For torus, nowadays all the properties in pi-base cannot distinguish it from S170 (Circle).